_visualisation/optional/annotation.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Annotation
+position: 2.01
+---
+
+Annotation is an [Optional Feature](/visualisation/optional) of BOVS Diagrams. It allows you to label Arrows to be more specific about the interest relationships they represent.
+{: .lead}
+
+
+* TOC
+{:toc}
+
+## Appearance
+
+![BOVS Diagram with Annotations](/visualisation/diagrams/bovs-optional-annotation.png){: style="width: 62.5%" }
+
+Annotations are text labels that appear alongside Arrows. They may appear at the start, middle or end of the Arrow; whatever is the clearest in each situation (this is usually the start). These labels are written in a standard way:
+
+* The nature of interest is given in the active voice, e.g. "Owns" or "Controls"
+* The level of interest is given as a percentage, e.g. "10%".
+
+Annotations respect the [Chirality](/visualisation/core/chirality) rule, that is:
+
+* Annotations about ownership appear on the left/below Arrows
+* Annotations about control appear on the right/above Arrows
+
+
+## Detail Level
+
+Annotations across your diagram must use a consistent detail level. Annotations may either *qualify* interests or additionally *quantify* them. In most cases, you should prefer to provide full detail, that is, to quantify interests.
+
+### Qualifying Interests
+
+Annotations indicate if the nature of interest is ownership, control or both; but they do not indicate the level of interest.
+
+![BOVS Diagram with Annotations Qualifying Interests](/visualisation/diagrams/bovs-optional-annotation-qualified.png){: style="width: 100.5%" }
+
+If your diagram also has an [Interest Focus](/visualisation/core/focus-depth), omit all annotations except those to mark Unknown or Unquantifiable interests (see below).
+
+### Quantifying Interests
+
+Annotations additionally indicate the level of ownership or control.
+
+![BOVS Diagram with Annotations Quantifying Interests](/visualisation/diagrams/bovs-optional-annotation-quantified.png){: style="width: 100%" }
+
+If your diagram also has an [Interest Focus](/visualisation/core/focus-depth), omit the nature of interest and write the level (percentage) only.
+
+![BOVS Diagram with Annotations and Interest Focus](/visualisation/diagrams/bovs-optional-annotation-interest-focus.png){: style="width: 75%" }
+
+
+## Relevance
+
+![BOVS Diagram showing Annotation applied only to relevant Arrows](/visualisation/diagrams/bovs-optional-annotation-relevance.png){: style="width: 100%" }
+
+All [relevant](/visualisation/core/relevance) Arrows in your diagram must be annotated.
+
+* Annotations should not be added to non-relevant Arrows.
+* Omission of an annotation (of a relevant Arrow) means absence of interest.
+  * e.g. if there is no annotation on the side of the Arrow where ownership information appears, it means there is no ownership interest.
+
+
+## Indirect Arrows
```

Comment:
This section should be moved into the core part of the specification, under the 'Interests (Arrows)' section.

Reply:
Yes that's where I originally intended to place this, but when writing I realised that – unless I'm missing something! – an Indirect Arrow has no meaning unless it is annotated. And so by that token can only be drawn if you are using Annotation. I think this just shows how important Annotation is i.e. despite being an "Optional" Feature it will nearly always be used (which is noted in the docs).

Reply:
It's true that it has little meaning unless it is annotated, however it is technically possible in BODS to record that there is an indirect interest between two parties without supplying any information its nature.

More importantly, when disclosing beneficial owners, it is always the ultimate and often indirect interests that are of primary concern. It's the case that intermediaries may not have to be disclosed at all. In this sense, indirect interests are a core feature that need to be put up front. Hope that makes sense!

Reply:
Thanks Kadie, I see! —

So for instance we could have BODS data saying simply "**Person A →indirect→ Entity B**" with no other statements (and this might even be quite common).

If we were to visualise this "verbatim" (as it were), we'd get a diagram containing only two Nodes connected by an Indirect Arrow. To me, such a diagram would not really be helpful. It also would break a number of BOVS rules (i.e. to allow this, we'd have to write in some new edge cases).

I would prefer to interpret that BOVS data – for the purpose of visualisation – as missing the statement "**Person A → 1 or more Unknown Parties → Entity B**". (Which is equivalent to "Person A →indirect→ Entity B" but restated in direct-relationship i.e. BOVS terms.)

Then you can draw it following the normal BOVS rules, and make a decision about whether, additionally, to draw an Indirect Arrow from the first to last Node (which you annotate either with numbers or '?', following the rules).

This raises a good general point that there will be many cases where BOVS cannot achieve a 1:1 correspondence with a BODS document. I see that as being "as intended", i.e. we didn't set out _purely_ to visualise BODS data, but rather to clarify situations visually. The important implication is that there is an interpretation/translation step between BODS/raw info and drawing a BOVS diagram.

So, returning to specifics, my preference is definitely to retain the current approach where Direct Arrows are the necessary backbone of the diagram, and Indirect Arrows are a secondary/optional feature – crucial as they are – with the expectation that in many cases, you will have to interpret an indirect BODS statement in terms of a direct BOVS Chain via an Unknown intermediary.

Let me know your thoughts!

Reply:
I'm in agreement with Virgil here - that when BODS data says: **Person A -> indirectly controls -> Company B**, what this means, and how it should be presented is: **Person A → 1 or more Unknown Parties → Entity B**. 

I can imagine in a whiteboard activity drawing simply a dotted indirect line whilst talking this through. but a full explanation should always show that this _means_ there is 1 or more unknown party in play. 

A prepared diagram for this could, I think, show both the '1 or more Unknown Parties' route, and the dotted indirect route **and** could put the '1 or more Unknown Parties' route at 25% transparency - to draw attention to the indirect dotted line.

Reply:
I agree with all that. In practice, though, disclosure often involves declaration of the aggregate/resolved interest(s) in a declaring company. These are effectively the 'properties' of an often indirect interest(s) and would be displayed as such. Therefore IMO we have to allow that it's a core feature because implementers need to know about it.

But if Tim's happy to leave it as an 'optional' feature and not put it up front - fine,

Reply:
@kd-ods I think we can still put it up front – I will see about adding more info on this to the Core page. So we surface even in this context that there is an important, though strictly optional, feature you ought also to know about. Hope that's the best of both worlds!

Reply:
@Convincible - great.